spec/requests/users_controller_spec.rb

```diff
@@ -31,6 +33,26 @@
         post api_v1_users_path, params: valid_user_params, headers: headers
         expect(User.find_by(email: valid_user_params[:user][:email]).role).to eq(role)
       end
+
+      context 'User language' do
+        before do
+          allow(I18n).to receive(:default_locale).and_return(:gl)
```

Comment:
I would move this into the second test since it is only needed for a specific test case

Reply:
But the other tests will then run with unmacked `I18n` instance that may default to the same language in the `user_params`?

Reply:
What about changing the language in the `user_params` to a none language Alpha2 code like 'language'?

Reply:
How would `I18n.default_locale` be set to anything?

Reply:
Rails will set it by default to `:en` if the `DEFAULT_LOCALE` isn't specified.
If the ENV VAR is specified then the `I18n.default_locale` will match it according to the `locales` initializer introduces in #3523.

Reply:
In that case, I would change the param coming in to be something that's not the standard default value `fr` or `es` should be fine

Reply:
Good idea, I'll change it